glambie/processing/path_handling.py

```diff
@@ -41,6 +41,20 @@ def get_region_output_folder_path(self, region: RGIRegion) -> str:
         Path(folder_path).mkdir(parents=True, exist_ok=True)
         return folder_path
 
+    def get_config_output_folder_path(self) -> str:
+        """
+        Returns requested folder path for saving configs of a run.
+        If a folder dosen't exist it is iteratively created on the fly.
+
+        Returns
+        -------
+        str
+            output folder path
+        """
+        folder_path = os.path.join(self.base_path, "configs")
```

Comment:
This is better off as a single line:

```python
return os.makedirs(os.path.join(self.base_path, "configs"), exist_ok=True)
```

Reply:
Likewise for other uses of the pattern `Path(folder_path).mkdir(parents=True, exist_ok=True)`

Reply:
done everywhere

Reply:
os.makedirs() does not return a path

Reply:
I've changed Path(folder_path).mkdir(parents=True, exist_ok=True) to os.makedirs(folder_path, exist_ok=True), not sure this makes any difference?

Reply:
you're correct, it doesn't make a huge amount of difference in the end. In any case, resolved.